driver/kubernetes/manifest/manifest.go

```diff
@@ -118,21 +119,25 @@ func NewDeployment(opt *DeploymentOpt) (d *appsv1.Deployment, c []*corev1.Config
 			Data: cfg.files,
 		}
 
-		d.Spec.Template.Spec.Containers[0].VolumeMounts = []corev1.VolumeMount{{
-			Name:      cfg.name,
-			MountPath: path.Join("/etc/buildkit", cfg.path),
-		}}
-
-		d.Spec.Template.Spec.Volumes = []corev1.Volume{{
-			Name: "config",
-			VolumeSource: corev1.VolumeSource{
-				ConfigMap: &corev1.ConfigMapVolumeSource{
-					LocalObjectReference: corev1.LocalObjectReference{
-						Name: cc.Name,
+		d.Spec.Template.Spec.Containers[0].VolumeMounts = []corev1.VolumeMount{
+			{
+				Name:      cfg.name,
+				MountPath: path.Join("/etc/buildkit", cfg.path),
+			},
+		}
+
+		d.Spec.Template.Spec.Volumes = []corev1.Volume{
+			{
+				Name: "config",
+				VolumeSource: corev1.VolumeSource{
+					ConfigMap: &corev1.ConfigMapVolumeSource{
+						LocalObjectReference: corev1.LocalObjectReference{
+							Name: cc.Name,
+						},
 					},
 				},
 			},
-		}}
+		}
```

Comment:
Seems unneeded change

Reply:
yep, sorry, disabled the auto reformat in the IDE :)